WcaOnRails/config/locales/da.yml

```diff
@@ -2698,7 +2991,7 @@ da:
         content:
           #original_hash: 024ce81
           '1': >-
-            Du har lov til at bruge magnetiske terninger så længe at det følger
+            Du har lov til at bruge magnetiske terninger så længe at det følger 
```

Comment:
This means you most likely have a double space between `følger` and the `<a>` tag and it makes the build fail. Could you please check and resubmit the update? =)